nginx/datadog_checks/nginx/nginx.py

```diff
@@ -41,6 +41,7 @@ def fromisoformat(ts):
 PLUS_API_STREAM_ENDPOINTS = {
     "stream/server_zones": ["stream", "server_zones"],
     "stream/upstreams": ["stream", "upstreams"],
+    "stream/zone_sync": ["stream", "zone_sync"],
```

Comment:
should this be added only if version 3 is selected?

Reply:
Yeah, but unsure how to only add this if plus_api_version >3

Reply:
In line 95:
```
self.plus_api_stream_endpoints = copy.deepcopy(PLUS_API_STREAM_ENDPOINTS)
if int(self.instance.get('plus_api_version')) >= 3:
    self.plus_api_stream_endpoints[ "stream/zone_sync"] = ["stream", "zone_sync"]
```

And then replace all usages of `PLUS_API_STREAM_ENDPOINTS` with self.plus_api_stream_endpoints

Reply:
I kinda like having all the target endpoints listed in the same location. Commit coming shortly that will skip if plus_api_version <3 and endpoint == /stream/zone_sync.